atc/api/pipelineserver/archive.go

```diff
@@ -0,0 +1,22 @@
+package pipelineserver
+
+import (
+	"net/http"
+
+	"github.com/concourse/concourse/atc/db"
+)
+
+func (s *Server) ArchivePipeline(pipelineDB db.Pipeline) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if !s.enableArchivePipeline {
+			http.Error(w, "endpoint is not enabled", http.StatusForbidden)
+			return
+		}
+		s.logger.Debug("archive-pipeline")
```

Comment:
Given that I find myself reading other people's logs, I feel motivated to make the system more verbose by default, just in case. This line is actually enforced by a unit test, because I don't want to rely on other people to turn on auditing if they're then going to be needing my advice on the state of their cluster.

Reply:
yeah, I do get the feeling :/ 

> I don't want to rely on other people to turn on auditing if they're then going to be needing my advice on the state of their cluster.

although this does rely on the fact that they turned `debug` on, right? (which, sure, can be turned on at runtime, which is quite useful, but is also super expensive to have it on AFAIR). 

if the same was true for audit (assuming the pain-point is quickly turning the system into a more verbose mode), would you think this log message in this particular endpoint would not be needed anymore?